swap/tests/harness/mod.rs

```diff
@@ -734,28 +743,15 @@ impl TestContext {
             .estimate_fee(TxRedeem::weight(), self.btc_amount)
             .await
             .expect("To estimate fee correctly");
-        self.alice_starting_balances.btc + self.btc_amount - fee
+        self.btc_amount - fee
     }
 
     fn bob_redeemed_xmr_balance(&self) -> monero::Amount {
         self.bob_starting_balances.xmr
     }
 
-    async fn bob_redeemed_btc_balance(&self, state: BobState) -> Result<bitcoin::Amount> {
-        self.bob_internal_bitcoin_wallet.sync().await?;
-
-        let lock_tx_id = if let BobState::XmrRedeemed { tx_lock_id } = state {
-            tx_lock_id
-        } else {
-            bail!("Bob in not in xmr redeemed state: {:?}", state);
-        };
-
-        let lock_tx_bitcoin_fee = self
-            .bob_internal_bitcoin_wallet
-            .transaction_fee(lock_tx_id)
-            .await?;
-
-        Ok(self.bob_starting_balances.btc - self.btc_amount - lock_tx_bitcoin_fee)
```

Comment:
If you are deleting code like this, can we

a) have a dedicated commit for it
b) explain why we can do it without causing problems?

Reply:
> a) have a dedicated commit for it

Not really, it depends on the other change in this commit. 

> b) explain why we can do it without causing problems?

The reason is that the balance of Bob's `personal` bitcoin wallet needs to be 0 after the bitcoin was redeemed. Can add this reasoning into the commit message

Reply:
> Not really, it depends on the other change in this commit.

It should be part of that commit then.

Reply:
>(Do the e2e tests even pass with the previous one but without this one?) 

Yes, because we were asserting something else: that the balance of the other wallet is `starting_balance - btc_amount - fee`.